packages/web-react/src/components/Toast/demo/ToastStaticToast.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button } from '../../Button';
+import Toast from '../Toast';
+import ToastBar from '../ToastBar';
+
+const ToastStaticToast = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const buttonLabel = isOpen ? 'Hide' : 'Show';
```

Comment:
⚠️ This behaves differently from the `web` implementation. The `hide` method should **remove** the Toast from the DOM so I'm no able to open it again.

Or have we decided to make it different in React? 🤔

Reply:
In React, it is removed from the DOM, but can be brought back, unlike in web. So the button does not have to be disabled. I can implement functionality to disable the button after click, but do we want that?

Reply:
As we discussed in the office, this state is okay, and we can leave it as it is.